packages/office-ui-fabric-react/src/components/HoverCard/HoverCard.Props.ts

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import { HoverCard } from './HoverCard';
+import { IRenderFunction } from '../../Utilities';
+import { IStyle, ITheme } from '../../Styling';
+
+export interface IHoverCard {
+
+}
+
+/**
+ * HoverCard component props.
+ */
+export interface IHoverCardProps extends React.HTMLAttributes<HTMLDivElement | HoverCard> {
```

Comment:
Can we control stickiness too (to not disappear even when you mouse out)? We'd love to use this and we need that support, as our LPC is sticky in SPhome and ODSP currently.